src/operator/windowToggle.ts

```diff
@@ -131,10 +131,16 @@ class WindowToggleSubscriber<T, O> extends OuterSubscriber<T, any> {
         const context = { window, subscription };
         this.contexts.push(context);
         const innerSubscription = subscribeToResult(this, closingNotifier, context);
```

Comment:
it is immediately unsubscribed when it's empty observable, so synchronously invoke `notifyComplete` and continue execution to line 135. This change's using those, check innersubscription status at that moment - if it's immediately unsubscribed, close window without setting context.